### PR TITLE
Add Subaccount error messages

### PR DESCRIPTION
**Pull request by marklewin**
# Description

* Added Subaccount error messages [DEVX-3457](https://nexmoinc.atlassian.net/browse/DEVX-3457)
* Based on list of errors provided [here](https://docs.google.com/document/d/1KFCAWZOpcKIxLTS3WVNqkRsFcyZ1HyVNg0MCe6wMLqs/edit?usp=sharing)

# Checklist

- [X] version number incremented (in the `info` section of the spec)

### CODE REVIEW

File: definitions/subaccounts.yml [hunk 1]
```diff
@@ -663,3 +663,23 @@ components:
         - from
         - to
         - amount
+
+x-errors:
```
Comment (tbedford) [inline definitions/subaccounts.yml:667]: I'll need some help with this one. From what I can make out this adds the following error messages:

1. Request failed due to validation errors
2. Invalid transfer
3. Maximum number of subaccounts reached
4. An error has occurred in the platform while processing the request.

However, when looking at the rendering, and clicking through the buttons for 400, 401,...., 422 I can only see the first in this list rendered, that is, 422 corresponds to 'Request failed due to validation errors'. I don't see where error messages 2, 3, 4  get rendered.

Comment (marklewin) [reply definitions/subaccounts.yml:667]: There doesn’t appear to be any correlation between the errors reported here and the API reference - check out the SMS errors for example. I think it’s more a case of ‘if you get a response with the following error text in it, here is something sensible to do’.

Comment (mheap) [reply definitions/subaccounts.yml:667]: For future reference, these errors show up on the `api-errors` page - see https://developer.nexmo.com/api-errors/subaccounts

Comment (tbedford) [reply definitions/subaccounts.yml:667]: ah! Thanks @mheap

**Pull request merged**